data_model/src/metadata.rs

```diff
@@ -122,6 +128,11 @@ impl Metadata {
         map.get(key)
     }
 
+    /// Return if the metadata map contains the given key.
```

Comment:
maybe check instead of return?

```suggestion
    /// Check if the metadata map contains the given key.
```

if you want to say return, then you need to say "return the Boolean value" or something along these lines